@neheb 
````
bootstrap: running: /__w/openwrt/openwrt/openwrt/staging_dir/host/share/gnulib/gnulib-tool --no-changelog --avoid=dummy --libtool --macro-prefix=GL --with-tests --tests-base=gnulib-tests --aux-dir=build-aux --m4-base=m4 --local-dir=gl --local-dir=gl-mod/bootstrap --import announce-gen bootstrap...
gnulib-tool: warning: module bootstrap doesn't exist
gnulib-tool: warning: module extract-trace doesn't exist
gnulib-tool: warning: module inline-source doesn't exist
====== Make errors from logs/tools/e2fsprogs/compile.txt ======
  : ; \
fi
make[4]: Leaving directory '/__w/openwrt/openwrt/openwrt/build_dir/host/e2fsprogs-1.47.0/po'
making install in scrub
make[4]: Entering directory '/__w/openwrt/openwrt/openwrt/build_dir/host/e2fsprogs-1.47.0/scrub'
	MKDIR_P /etc/cron.d
	MKDIR_P /__w/openwrt/openwrt/openwrt/staging_dir/host/lib/e2fsprogs
	INSTALL /etc/cron.d/e2scrub_all
	MKDIR_P /__w/openwrt/openwrt/openwrt/staging_dir/host/bin /__w/openwrt/openwrt/openwrt/staging_dir/host/share/man/man8 /__w/openwrt/openwrt/openwrt/staging_dir/host/etc
/__w/openwrt/openwrt/openwrt/staging_dir/host/bin/install: cannot remove '/etc/cron.d/e2scrub_all': Permission denied
make[4]: *** [Makefile:451: install-crond] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: Leaving directory '/__w/openwrt/openwrt/openwrt/build_dir/host/e2fsprogs-1.47.0/scrub'
make[3]: *** [Makefile:444: install-progs-recursive] Error 1
make[3]: Leaving directory '/__w/openwrt/openwrt/openwrt/build_dir/host/e2fsprogs-1.47.0'
make[2]: *** [Makefile:51: /__w/openwrt/openwrt/openwrt/staging_dir/host/stamp/.e2fsprogs_installed] Error 2
````

--- a/tools/e2fsprogs/Makefile
+++ b/tools/e2fsprogs/Makefile
@@ -33,6 +33,10 @@ HOST_CONFIGURE_ARGS += \
 	--disable-fuse2fs \
 	--with-crond-dir=no
 
+# The following uses pkg-config the wrong way around. Just override it.
+HOST_CONFIGURE_VARS += \
+	ac_cv_lib_uuid_uuid_generate=yes
+
 define Host/Prepare
 	$(call Host/Prepare/Default)
 	rm -rf $(HOST_BUILD_DIR)/doc

--- a/tools/util-linux/patches/110-pkgconfig-static.patch
+++ b/tools/util-linux/patches/110-pkgconfig-static.patch
@@ -1,0 +1,24 @@
+From 81ea117e0396274808a77d40bf11af44ea047285 Mon Sep 17 00:00:00 2001
+From: Rosen Penev <rosenp@gmail.com>
+Date: Mon, 23 Sep 2024 12:01:40 +0200
+Subject: [PATCH] pkgconfig: adjust for static libraries
+
+OpenWrt builds only static libuuid for tools. Adjust the pkgconfig file
+accordingly.
+
+Addresses: https://github.com/util-linux/util-linux/issues/3210
+Signed-off-by: Rosen Penev <rosenp@gmail.com>
+---
+ libuuid/uuid.pc.in | 2 +-
+ 1 file changed, 1 insertions(+), 2 deletions(-)
+
+--- a/libuuid/uuid.pc.in
++++ b/libuuid/uuid.pc.in
+@@ -6,6 +6,5 @@ includedir=@includedir@
+ Name: uuid
+ Description: Universally unique id library
+ Version: @LIBUUID_VERSION@
+-Requires:
+ Cflags: -I${includedir}/uuid
+-Libs: -L${libdir} -luuid
++Libs: -L${libdir} -luuid -lpthread

